NEWS

```diff
@@ -57,7 +57,39 @@ included in the vX.Y.Z section and be denoted as:
 --------------------
 
 Bug fixes/minor improvements:
+
 - Remove IB XRC support from the OpenIB BTL due to lack of support.
+- Fix a problem with MPI_IALLTOALLW when using zero-length messages.
```

Comment:
@jsquyres let's clarify.  its not completely broken.  it works for me on my connectx5 MOFED 4.0.0 cluster.  It just doesn't work for Mellanox.